.vscode/launch.json

```
@@ -9,7 +9,8 @@
             "program": "${workspaceFolder}/cmd/kyverno",
             "args": [
                 "--kubeconfig=${userHome}/.kube/config",
-                "--serverIP=<local ip>:9443",
+                "--serverIP=<SERVER-IP>:9443",
+                "--v=2",
```

Were these added again?

---

Previously it had my local IP....changed to reflect its a var and added v2 to match defaults. Any issues with this?

---

It's a single dash flag.
```suggestion
                "--serverIP=<SERVER-IP>:9443",
                "-v=2",
```